Dockerfile

```diff
@@ -1,14 +1,38 @@
 FROM python:3.8-buster
 
-RUN pip install --upgrade pip
+ARG THIS_USER="packpacker"
```

Comment:
most of the user stuff is lifted/adapted from the jupyter notebook dockerfile. basically, we have to make ourselves a user and group and be sure to chown any directories we want that user to be able to use